tdrs-backend/tdpservice/users/test/test_api/test_permissions.py

```diff
@@ -26,9 +26,9 @@ def test_permission_list_not_found(api_client, create_test_users):
 
 # @pytest.mark.django_db
 # def test_permission_list_unauthorized(api_client, create_test_users):
-#     """Test data prepper does not have access."""
+#     """Test data analyst does not have access."""
```

Comment:
Not for this PR but we should definitely address these commented out tests with #888